### PR TITLE
Some new actions! :)

### PR DESCRIPTION
**Pull request by tallguyjenks**

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Oh awesome! I will have a look through later on and have a play, thank you @tallguyjenks - the YouTube one definitely looks super interest 💪

**Comment by tallguyjenks** [comment]
@eddiejaoude if you want to see it live I have the article, youtube, and GH activity all on my profile repo https://github.com/tallguyjenks/tallguyjenks

**Comment by eddiejaoude** [comment]
Thanks @tallguyjenks 👍

**Comment by eddiejaoude** [comment]
Stalebot lets keep this for now please 🤓

**Review by dumbcoder7 — APPROVED** [review]

**Comment by eddiejaoude** [comment]
Not sure we want all these changes just yet. We can bring them in as we use them

**Comment by tallguyjenks** [comment]
@eddiejaoude still blocked? lol i have a few more actions too now there's so many good ones

**Comment by eddiejaoude** [comment]
hey @tallguyjenks I am blocked on other stuff and this one is slowly moving down my list 😢 . Oh feel free to append the new actions to this PR?

**Comment by krishguptadev** [comment]
Since this PR has not been updated/discussed in a while, I am curious why didn't Stale bot interfere.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name-template: "$RESOLVED_VERSION"
```
Comment (krishguptadev) [inline .github/release-drafter.yml:1]: We would rather prefer to use this action [here](https://github.com/EddieHubCommunity/LinkFree/blob/main/.github/workflows/release.yml)

File: .github/workflows/articles.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: <NAME> on Medium
```
Comment (krishguptadev) [inline .github/workflows/articles.yml:1]: Why do we need this? We don not have a blog that supports RSS for now, and if it were, I definitely wouldn't add them to a Action's github readme

File: .github/workflows/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: Release Drafter
```
Comment (krishguptadev) [inline .github/workflows/release-drafter.yml:1]: https://github.com/EddieHubCommunity/gh-action-community/pull/56/files#r753748836

File: .github/workflows/update-readme.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: Update README
```
Comment (krishguptadev) [inline .github/workflows/update-readme.yml:1]: Why would we need to add someone's github activity to our Readme?

File: .github/workflows/youtube-workflow.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+name: Latest YouTube Videos
```
Comment (krishguptadev) [inline .github/workflows/youtube-workflow.yml:1]: Also, why would we wanna add youtube videos on the readme of a github action

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by krishguptadev** [comment]
@tallguyjenks From the PR pattern I think you've made the PR to the wrong repo, this repo is about GitHub Actions and you've added stuff relevant to a github profile readme.

@eddiejaoude I think this PR can be closed 🙂

**Comment by tallguyjenks** [comment]
they were just example actions for others to grab and use, not something I meant to actually run on the repo. by itself, the things affecting the readme wouldn't function without the right HTML comments placed in the readme anyhow.

**Comment by krishguptadev** [comment]
> they were just example actions for others to grab and use, not something I meant to actually run on the repo. by itself, the things affecting the readme wouldn't function without the right HTML comments placed in the readme anyhow.

but they shouldn't be in .github/workflows any ways 😓

**Pull request closed without merging**